### PR TITLE
Implement image-live. Write tests for image-live

### PR DESCRIPTION
**Pull request by sherryy**
implement wikibase-term IMAGE-LIVE attribute natively in WikipediaBase

Fixes: #100

### CODE REVIEW

File: wikipediabase/resolvers/term.py [hunk 1]
```diff
@@ -51,6 +52,28 @@ def coordinates(self, article, _, **options):
...
+            if href:
+                if img in href and "File" not in href:
+                    img_url = href
+                    break
+        if "size" in options:
+            size = options["size"]
+            insert_to = re.search("/\w/\w\w/", img_url).start()
+            img_url = img_url[:insert_to] + "/thumb" + img_url[insert_to:] + "/" + str(size) + "px-" + img
+        best["file"] = "https:" + img_url
+        return best
```
Comment (chsamlee) [inline wikipediabase/resolvers/term.py]: You should always return a list of dicts.

**Comment by chsamlee** [comment]
LGTM.

File: wikipediabase/resolvers/term.py [hunk 2]
```diff
@@ -51,6 +52,28 @@ def coordinates(self, article, _, **options):
...
+        if not imgs:
+            return []
+        best = imgs[0]
+        img = best["file"]
+        file_page = get_fetcher().html_source("File:" + img)
+        soup = BeautifulSoup(file_page, 'html.parser')
+        for hyperlink in soup.find_all('a'):
+            href = hyperlink.get('href')
+            if href:
+                if img in href and "File" not in href:
```
Comment (alvaromorales) [inline wikipediabase/resolvers/term.py]: what happens when img is HTML escaped?  For example:

```python
self.term_resolver.image_data("Thoughts of Love", "image-data")
```

returns `"TOL%28Custom%29.jpg"`. Will you find `"TOL%28Custom%29.jpg"` in `href`, or do you need to unescape?

Before line 70 you might need to check if img_url is `None` to be safe?

File: wikipediabase/resolvers/term.py [hunk 3]
```diff
@@ -51,6 +52,28 @@ def coordinates(self, article, _, **options):
...
+                    img_url = href
+                    break
+        if "size" in options:
```
Comment (alvaromorales) [inline wikipediabase/resolvers/term.py]: @chsamlee do you lowercase the keys of `options`? What if someone calls `(get "wikibase-term" "Foo" (:code "IMAGE-LIVE") (:SIZE 100))`?

Comment (chsamlee) [inline wikipediabase/resolvers/term.py]: I believe so (frontend.py, line 47)

**Comment by alvaromorales** [comment]
lgtm :ship:

**Pull request merged**